docker/001-config.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+set -ex
+
+echo "IPFS PATH: ${IPFS_PATH}"
+
+# We backup old config file
+cp ${IPFS_PATH}/config ${IPFS_PATH}/config_bak
+
+# We inject the S3 plugin datastore
+# Important: Make sure your fill out the optionnal parameters $CLUSTER_S3_BUCKET, $CLUSTER_AWS_KEY, $CLUSTER_AWS_SECRET in the cloudformation parameters
+cat ${IPFS_PATH}/config_bak | \
+jq ".Swarm.ResourceMgr = {}" | \
+jq ".Swarm.ConnMgr = {}" | \
```

Comment:
```suggestion
```

Given that this is just using the kubo defaults you shouldn't need to explicitly set anything here.